### DISCUSSION TITLE
Is it possible to enable AVI video format?

### DISCUSSION THREAD

**Discussion by b00st3d** [mediacms-io/mediacms#831] [Q&A] [answered]
After uploading an AVI video file, I try to play the file and it says "No compatible source was found for this media."  When looking at the "My media" tab, I can see that the file uploads as audio.  When looking in the Admin control panel, I can see that the only extensions recognized are mp4, webm, and gif.  Is there some way to add more file extensions to this?  AVI is important because that is the format that my kids' cameras record in.

I have built MediaCMS as a server and as a docker container thinking I did something wrong.

**Comment by mgogoulos** [comment]
Hi, 
this is definitely a bug, as AVI was supported for sure! And I can also say I have lots of AVIs from the "recent" past ;)


So I see this support broke 1 year ago on https://github.com/mediacms-io/mediacms/pull/505/files
https://github.com/mediacms-io/mediacms/issues/213

**Comment by mgogoulos** [comment, marked as answer]
AVIs are incorrectly marked as 'audio' after a fix on mp3/flac files with an image (that were falsely considered to be videos). 

I just opened a PR that should add an extra check and not make the mistake for AVIs: https://github.com/mediacms-io/mediacms/pull/833

I tested it locally and it works. If you want to test it, you can checkout to the specific branch, and restart web and celery_worker containers. 

Then you have to trigger somehow the handling of the existing videos, best way would be to delete these files and re-upload, hope this is possible to do! If I get a confirmation that this works too for you, I will merge this PR. 

Cheers

**Reply by b00st3d** [reply]
I just switched to the fix-avi branch and all seems to be working as expected
